pkg/app/master/inspectors/container/container_inspector.go

```diff
@@ -874,16 +874,12 @@ func (i *Inspector) RunContainer() error {
 	}
 
 	if i.PrintState {
-		var containerIP string
-		if i.ContainerInfo.NetworkSettings != nil {
-			containerIP = i.ContainerInfo.NetworkSettings.IPAddress
-		}
 		i.xc.Out.Info("container",
 			ovars{
 				"status": "running",
 				"name":   containerInfo.Name,
 				"id":     i.ContainerID,
-				"ip":     containerIP,
+				"ip":     i.ContainerInfo.NetworkSettings.IPAddress,
```

Comment:
there are cases when `i.ContainerInfo.NetworkSettings` is nil, so this will panic in those cases (not common, but it does happen)

Reply:
There’s a guard against this possibility [above](https://github.com/docker-slim/docker-slim/blob/2c19aa4f8ac63306e78c5d821a6c19d12ed46aea/pkg/app/master/inspectors/container/container_inspector.go#L844).

Reply:
good catch :)